probes/pkg/ping/server/server.go

```diff
@@ -50,7 +50,9 @@ func Run(config *PingServerConfig) {
 }
 
 func pong(w http.ResponseWriter, r *http.Request) {
-	klog.V(4).Infof("pong -> %s\n", r.RemoteAddr)
+	// dereference r so it can be closed by http framework
+	addr := r.RemoteAddr
```

Comment:
I don't understand why this is needed.

Reply:
+1, I don't get it either

Reply:
Removed. I am pretty sure it helped in tests yesterday, but now I don't see it.